Bottom: 2976fa73133e0a1bc27b2680b0772a8ac0bd13b2
Top:    6277ae55349fe63894717a51ca838498bf6e04b7
Author: Alexey Lyashkov <shadow@Alexeys-MacBook-Pro.local>
Date:   2016-01-18 11:00:47 +0300

Refresh of sock-resource.patch

---

diff --git a/ib-sock-int.h b/ib-sock-int.h
index 4e9988d..9d4cd30 100644
--- a/ib-sock-int.h
+++ b/ib-sock-int.h
@@ -25,11 +25,6 @@ struct ib_sock_mem {
 	/* memory window to map.
 	 * all ? or most cards may work with single == global MR  */
 	struct ib_mr		*ism_mr;
-	
-	/* # send work items for single transfer */
-	unsigned		ism_wr_count;
-	/* ...and their memory */
-	unsigned		ism_sge_count;
 };
 
 struct IB_SOCK {
